### ISSUE TITLE
Improper codeblock in README

### ISSUE THREAD

**Issue by pri22296** [fossasia/yaydoc#103]
### This is a
- [x] Bug report
- [ ] Feature request

### Current behaviour

The **Using SSH** section renders in https://yaydoc.fossasia.org as follows:
![screenshot 142](https://user-images.githubusercontent.com/11555869/27011693-08e36312-4edf-11e7-86ab-86467d6e8a2b.png)

### Expected behaviour

It should render like as follows
![screenshot 143](https://user-images.githubusercontent.com/11555869/27011702-3a090744-4edf-11e7-949a-034ddb33a6af.png)

Related to jgm/pandoc#3733 (Incorrect parsing by pandoc's GFM reader)

**Comment by pri22296** [comment]
This issue is due to jgm/pandoc#3733 (Incorrect parsing by pandoc's GFM reader). Leaving this issue open till it's not fixed by pandoc

**Comment by imujjwal96** [comment]
@pri22296 https://github.com/jgm/pandoc/issues/3733 is closed. What's the status?

**Comment by pri22296** [comment]
Yeah, but It will be while before pandoc 2.0 is released where this change will be included.
